UnrealEngine.gitignore

```diff
@@ -1,6 +1,9 @@
 # Visual Studio 2015 user specific files
 .vs/
 
+# Visual Studio Code user specific files
+.vscode/
```

Comment:
For the moment I want to drop this and see where this discussion ends up, because VSCode also gets a mention there: https://github.com/github/gitignore/pull/3218/files#r347336080

Reply:
Unreal generates those files which contain absolute paths to the project and Unreal Engine directories.

![2020-07-07 09_32_40-tasks json -  vscode - Visual Studio Code](https://user-images.githubusercontent.com/2692096/86749948-87101800-c035-11ea-83fc-b4d838097d78.png)